react/FilePicker/queries.js

```diff
@@ -0,0 +1,36 @@
+import { Q, fetchPolicies } from 'cozy-client'
+
+const FILES_DOCTYPE = 'io.cozy.files'
+
+const defaultFetchPolicy = fetchPolicies.olderThan(30 * 1000)
+const TRASH_DIR_ID = `${FILES_DOCTYPE}.trash-dir`
+
+export const buildCurrentFolderQuery = folderId => ({
+  definition: () => Q(FILES_DOCTYPE).getById(folderId),
+  options: {
+    as: `buildCurrentFolderQuery-${folderId}`,
+    fetchPolicy: defaultFetchPolicy
+  }
+})
+
+export const buildContentFolderQuery = dirId => ({
+  definition: () =>
+    Q(FILES_DOCTYPE)
+      .where({
+        dir_id: dirId,
+        type: { $gt: null },
+        name: { $gt: null }
+      })
+      .partialIndex({
+        _id: {
+          $ne: TRASH_DIR_ID
+        }
+      })
+      .indexFields(['dir_id', 'type', 'name'])
+      .sortBy([{ dir_id: 'asc' }, { type: 'asc' }, { name: 'asc' }])
+      .limitBy(5),
```

Comment:
why 5?

Reply:
let's name it same way than in drive? https://github.com/cozy/cozy-drive/blob/master/src/drive/web/modules/queries.js#L62

Reply:
> why 5?

Why not? :P
More seriously, it was more for testing, what do you think about a limit between 10 and 20, so that the small connections have a quick result ?

Reply:
Let's use 20 in order to fill the gap.